mmaction/datasets/pipelines/augmentations.py

```diff
@@ -42,6 +42,210 @@ def _init_lazy_if_proper(results, lazy):
         assert 'lazy' not in results, 'Use Fuse after lazy operations'
 
 
+@PIPELINES.register_module()
+class Imgaug(object):
+    """Imgaug augmentation.
+
+    Adds custom transformations from imgaug library.
+    Please, visit `https://imgaug.readthedocs.io/en/latest/index.html`
+    to get more information. An example of ``transforms`` could be found
+    in `_default_transforms`
```

Comment:
If `_default_transforms` is intened to be visible from outside, then the leading `_` should be removed from the name